data/talkactions/scripts/looktype.lua

```diff
@@ -1,15 +1,36 @@
+local minLookType = 0
+local maxLookType = 699
+local blockLookType = {161, 162, 163, 164, 165, 166, 167, 168, 
+                          169, 170, 171, 172, 173, 174, 175, 176, 
+                          177, 178, 179, 180, 181, 182, 183, 184, 
+                          185, 186, 187, 188, 189, 190, 191, 474, 
+                          475, 476, 477, 478, 479, 480, 481, 482, 
+                          483, 484, 485, 638, 639, 640, 641, 642, 
+                          643, 645, 646, 652, 653, 654, 655, 656, 
+                          657, 658, 659, 660, 661, 662, 663, 135, 
+                          411, 415, 424, 439, 440, 468, 469, 518, 
+                          519, 520, 524, 525, 536, 543, 549, 576, 
+                          581, 582, 597, 616, 623, 635, 1}
+
+
 function onSay(player, words, param)
-	if not player:getGroup():getAccess() then
-		return true
-	end
-
-	local lookType = tonumber(param)
-	if lookType >= 0 and lookType ~= 1 and lookType ~= 135 and lookType ~= 411 and lookType ~= 415 and lookType ~= 424 and (lookType <= 160 or lookType >= 192) and lookType ~= 439 and lookType ~= 440 and lookType ~= 468 and lookType ~= 469 and (lookType < 474 or lookType > 485) and lookType ~= 501 and lookType ~= 518 and lookType ~= 519 and lookType ~= 520 and lookType ~= 524 and lookType ~= 525 and lookType ~= 536 and lookType ~= 543 and lookType ~= 549 and lookType ~= 576 and lookType ~= 581 and lookType ~= 582 and lookType ~= 597 and lookType ~= 616 and lookType ~= 623 and lookType ~= 625 and (lookType <= 637 or lookType >= 644) and (lookType <= 644 or lookType >= 647) and (lookType <= 651 or lookType >= 664) and lookType <= 699 then
-		local playerOutfit = player:getOutfit()
-		playerOutfit.lookType = lookType
-		player:setOutfit(playerOutfit)
-	else
-		player:sendCancelMessage("A look type with that id does not exist.")
-	end
-	return false
+    if not player:getGroup():getAccess() then
+        return true
+    end
+
+    local lookType = tonumber(param)
+
+    if not lookType then
+        player:sendCancelMessage("Param incorrect.")
+        return false 
+    end
+
+    if lookType >= minLookType and lookType <= maxLookType and not isInArray(blockLookType, lookType) then
```

Comment:
You definitely don't need this min and max look type. Everything inside the array is between the boundaries, so the check is redundant.